packages/components/input/src/components/osds-input/constants/input-size.ts

```diff
@@ -1,10 +0,0 @@
-enum ODS_INPUT_SIZE {
```

Comment:
keep exporting `ODS_INPUT_SIZE` as a wrapper of `ODS_COMMON_FIELD_SIZE`
same for `ODS_INPUT_TYPE`
(see text component constants for example)

the goal is to provide those to the end user, but not to provide any direct reference to ods-common-core

Reply:
Done